cargo-concordium/src/main.rs

```diff
@@ -768,28 +784,39 @@ fn handle_run_v1(run_cmd: RunCommand, module: &[u8]) -> anyhow::Result<()> {
     };
 
     // get the module schema if available.
-    let module_schema_opt = if let Some(schema_path) = &runner.schema_path {
-        let bytes = fs::read(schema_path).context("Could not read schema file.")?;
-        let schema = from_bytes(&bytes)
-            .map_err(|_| anyhow::anyhow!("Could not deserialize schema file."))?;
-        Some(schema)
-    } else {
-        let res = utils::get_embedded_schema_v1(module);
-        if let Err(err) = &res {
-            eprintln!(
-                "{}",
-                WARNING_STYLE.paint(format!(
-                    "Could not use embedded schema: {}.\nPlease provide a path to a valid schema.",
-                    err
-                ))
-            );
+    let module_schema_opt = match (&runner.schema_path, &runner.schema_v1_path) {
+        (Some(schema_path), _) => {
+            let bytes = fs::read(schema_path).context("Could not read schema file.")?;
+            let schema = from_bytes(&bytes)
+                .map_err(|_| anyhow::anyhow!("Could not deserialize schema file."))?;
+            Some(schema)
+        }
+        (_, Some(schema_v1_path)) => {
+            let bytes = fs::read(schema_v1_path).context("Could not read schema file.")?;
+            let schema = from_bytes(&bytes)
+                .map_err(|_| anyhow::anyhow!("Could not deserialize schema file."))?;
+            Some(VersionedModule::V0(schema))
+        }
+        _ => {
```

Comment:
Please document that this is the (None, None) case, the (Some, Some) being impossible.